.github/scripts/init-pre-release.sh

```diff
@@ -9,7 +9,6 @@ if [ -f ./configure ]; then
     git archive --prefix otp/ -o otp_src.tar.gz HEAD
```

Comment:
I think it was I who added this `if`-statement when I committed the `configure` scripts into the repo, but I'm not sure why. This line should be the only one needed (apart the #!/bin/sh line) in this file since `./configure` always should be present. Either I'm missing something now or I missed something then...

Reply:
Fixed. Removed `if `and entire `else` block.